newsroom/wire/search.py

```diff
@@ -83,6 +102,9 @@ def set_product_query(query, company):
         for product in products:
             if product.get('query'):
                 query['bool']['must'][0]['bool']['should'].append(_query_string(product['query']))
+    else:
+        # user does not belong to a company so blocking all stories
+        query['bool']['must'].append({'bool': {'must': [{'terms': {'products.id': [-1]}}]}})
```

Comment:
I think doing `abort(403)` might be better - we can handle it on client and display some message.